public/views/preferencesUnit.html

```diff
@@ -1,7 +1,7 @@
 <div 
   class="topbar-container" 
   ng-include="'views/includes/topbar.html'"
-  ng-init="titleSection='Unit'; goBackToState = true; noColor = true">
+  ng-init="titleSection='Unit for bytes'; goBackToState = null; noColor = true">
```

Comment:
just checking, no side effects from `goBackToState = null`? e.g. in Back navigation and for Back button on android?

Reply:
to make navigation consistent, i think it should show back button if user got to this page from settings

Reply:
Then it probably needs to be converted into modal because I couldn't figure out how to keep the back button work the same as choosing the unit size.
Back button has this `$root.go('^', true);`, but it was always taking it back to preferences even when I entered from Send screen.

Reply:
what defines `$root.go('^', true);` behavior and why it makes preferencesUnit go back to preferencesGlobal with that action? is it defined somewhere?

Reply:
Figured it out how `$root.go('^', true);` works and I made back button to go to walletHome (if entered from Send screen, global preferences works the same as before), but couldn't take it to Send screen itself because it seems there is no route for Send screen.